doc/cffi-manual.texinfo

```diff
@@ -6077,6 +6077,12 @@ A feature expression.
 @item library
 A library designator.
 
+@item canary
+A string denoting a foreign symbol that will be searched in core
+before attempting to load the library. If that symbol is found, the
+library is assumed to be statically linked and
+@code{load-foreign-library} only marks the library as loaded.
```

Comment:
Who would use this? Wrappers aren't defined via define-foreign-library.

Reply:
I compiled an SBCL that statically links libfixposix and openssl (https://cdn.cddr.org/ci/sbcl-2.0.4+r02-binary.tar.bz2), and I'd like to start using that: https://github.com/sionescu/iolib/commit/3bf5fc6883c8c2705744bf2d42522469c3f14cb7.

Reply:
It'd be nice to link somewhere (do SBCL or ECL or something document this?) or add a footnote about how one might statically link a library like this.

Reply:
I added a note to that effect, but I don't think we should add links because they can get stale and SBCL's is effectively undocumented even though supported.

Reply:
We could also use this for wrappers, but I don't think it's the right moment. We need a more comprehensive static linking story that supports lots of use cases, and that takes much work.

Reply:
Agreed.